src/Header/Header.js

```diff
@@ -4,6 +4,9 @@ const Header = () => {
   return (
```

Comment:
I do think it makes sense to have header as a functional component. @lo-la-do-li

Reply:
Agree - It would clean up our render in App by modularizing these components